backend/.rubocop.yml

```diff
@@ -44,6 +44,12 @@ Metrics/BlockLength:
 Metrics/ModuleLength:
   CountAsOne: ['heredoc']
 
+Metrics/CyclomaticComplexity:
+  Max: 8
+
+Metrics/PerceivedComplexity:
+  Max: 9
+
```

Comment:
on second though, let's not bump this up and instead add comments to turn it off & on for the block that's triggering it.  That way we're providing a clear rational in the code for "yes we know this is a bit complex and we decided it's ok", whereas doing it here leaves the reader guessing as to why we decided to increase it

Reply:
Ok will do.